packages/next/src/server/config.ts

```diff
@@ -923,8 +923,8 @@ function assignDefaults(
   // TODO(jiwon): remove once we've made new UI default
   // Enable reactOwnerStack when newDevOverlay is enabled to have
   // better call stack output in the new UI.
-  if (process.env.__NEXT_EXPERIMENTAL_NEW_DEV_OVERLAY === 'true') {
-    result.experimental.newDevOverlay = true
+  if (process.env.__NEXT_EXPERIMENTAL_NEW_DEV_OVERLAY === 'false') {
```

Comment:
does this mean on CI PPR tests doesn't run with new overlay?

Reply:
yea, just so we can still have some coverage on the old forked code until we remove it